jupyter-notebooks/analytics/user-guide/1. Getting Started with the Planet Analytics API.ipynb

```diff
@@ -1,9 +1,25 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup CSS Styles\n",
+    "from styles.plnbstyles import style_nb\n",
+    "style_nb(\"./styles/styles.css\")\n"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "<div id=\"ps-header\">\n",
```

Comment:
I don't think we need these divs.

Reply:
These hook into the custom styling from the CSS.

Reply:
I've removed the custom CSS styling and extraneous HTML elements.